controllers/account.js

```diff
@@ -82,8 +83,17 @@ module.exports.updateProfile = async (req, res) => {
     realName: req.body.realName,
     location: req.body.location,
     bio: req.body.bio,
+    mediaLinks: {},
```

Comment:
I'm not sure you need to have an empty object here since you're gonna override it anyway.

Reply:
What this does is that in case the user decides to remove their media account links, they simply delete their account link from the normally auto-filled form field. The entire mediaLinks object is set to be empty initially and is simply filled with the links that are present in the form.